### DISCUSSION TITLE
Verifying Geometry

### DISCUSSION THREAD

**Discussion by mariuszhermansdorfer** [VerifyTests/Verify#944] [Q&A]
Hello,

I'm developing a geometry processing plugin. On a very high level it consists of a UI layer, a few native libraries doing the underlying computation and a c# interface with a bunch of helpers to connect these two layers. To marshal data between individual modules I convert them to float arrays representing individual points and connections between them. These arrays can easily have 100.000 - 1.000.000 entries.

Do you have any suggestions on how to best structures my tests with verify? Should it be operating on raw arrays, or external files where the original data is saved to?

Any pointers in the right direction are much appreciated.

**Comment by SimonCropp** [comment]
i think it is better to pass the in mem data to verify.
